lib/blob/blobstore.c

```diff
@@ -817,6 +821,7 @@ blob_parse_page(const struct spdk_blob_md_page *page, struct spdk_blob *blob)
 				if (desc_extent->cluster_idx[i] != 0) {
 					blob->active.clusters[blob->active.num_clusters++] = bs_cluster_to_lba(blob->bs,
 							desc_extent->cluster_idx[i]);
+					blob->active.num_allocated_clusters++;
```

Comment:
Same here. Why need to update the value?

Reply:
Same as above